core/src/main/resources/xml/xsl/adapter2mermaid.xsl

```diff
@@ -0,0 +1,1073 @@
+						<forward name="emptyForwardName" path="{$emptyForwardName}"/>
+					</xsl:if>
+					<xsl:variable name="thenForwardName" select="@thenForwardName"/>
+					<xsl:if test="string-length($thenForwardName)&gt;0 and (forward/@name=$thenForwardName)=false()">
+						<forward name="thenForwardName" path="{$thenForwardName}"/>
+					</xsl:if>
+					<xsl:variable name="elseForwardName" select="@elseForwardName"/>
+					<xsl:if test="string-length($elseForwardName)>0 and (forward/@name=$elseForwardName)=false()">
+						<forward name="elseForwardName" path="{$elseForwardName}"/>
+					</xsl:if>
+				</xsl:when>
+				<xsl:otherwise>
+					<xsl:if test="not(forward[@name='success'])">
+						<forward name="success">
+							<xsl:attribute name="path">
+								<xsl:choose>
+									<xsl:when test="count($pipeline//pipeline/pipe)&gt;$pos">
+										<xsl:value-of select="$pipeline//pipeline/pipe[$pos+1]/@name"/>
+									</xsl:when>
+									<xsl:otherwise>
+										<xsl:value-of select="$pipeline//exit[@state='success']/@path"/>
+									</xsl:otherwise>
+								</xsl:choose>
+							</xsl:attribute>
+						</forward>
+					</xsl:if>
+				</xsl:otherwise>
+			</xsl:choose>
+		</xsl:variable>
+
+<!--		<forwardTest01>-->
+<!--			<xsl:copy-of select="$initialForwards1"/>-->
+<!--		</forwardTest01>-->
+
+		<xsl:variable name="initialForwards2">
+			<xsl:copy-of select="$initialForwards1/*"/>
+			<xsl:copy-of select="$pipeline/global-forwards/forward"/>
+			<xsl:if test="count($initialForwards1/*) = 0 and @className = 'nl.nn.adapterframework.pipes.XmlSwitch'">
+<!--				<isXmlSwitchWithNoForwards/>-->
+				<xsl:variable name="originPipeName" select="@name"/>
+				<xsl:for-each select="$pipeline//pipe[@name != $originPipeName]">
+					<xsl:variable name="pipeName" select="@name"/>
+<!--					<pipe name="{$pipeName}">-->
+<!--						<test01><xsl:value-of select="-->
+<!--										if($pipeline//pipeline/@firstPipe) then (-->
+<!--											$pipeline//pipeline/@firstPipe != $pipeName-->
+<!--										) else (-->
+<!--											$pipeline//pipeline/pipe[1]/@name != $pipeName-->
+<!--										)"/></test01>-->
+<!--						<test02><xsl:value-of select="count($pipeline//pipe/forward[@path = $pipeName]) = 0"/></test02>-->
+<!--						<test03><xsl:value-of select="count($pipeline//inputValidator/forward[@path = $pipeName]) = 0"/></test03>-->
+<!--						<test04><xsl:value-of select="count($pipeline//outputValidator/forward[@path = $pipeName]) = 0"/></test04>-->
+<!--						<test05><xsl:value-of select="count($pipeline//pipe[@notFoundForwardName = $pipeName]) = 0"/></test05>-->
+<!--						<test06><xsl:value-of select="count($pipeline//pipe[@emptyForwardName = $pipeName]) = 0"/></test06>-->
+<!--						<test07><xsl:value-of select="count($pipeline//pipe[@thenForwardName = $pipeName]) = 0"/></test07>-->
+<!--						<test08><xsl:value-of select="count($pipeline//pipe[@elseForwardName = $pipeName]) = 0"/></test08>-->
+					<xsl:if test="
```

Comment:
Begrijp ik deze test goed dat de XmlSwitch lijntjes krijgt naar alle pipes waar geen forwards naar verwijzen? Ik zou denken dat zo een lijst wel iets makkelijker te testen zou zijn, zonder alle pipes via een for-each af te lopen